Runtime/Core/SelectionPickerSettings.cs

```diff
@@ -74,9 +97,9 @@ public bool Equals(SceneSelection other)
             if (ReferenceEquals(this, other)) return true;
             return Equals(gameObject, other.gameObject)
                 && Equals(mesh, other.mesh)
-                && vertex == other.vertex
-                && edge.Equals(other.edge)
-                && Equals(face, other.face);
+                && Enumerable.SequenceEqual(vertexes, other.vertexes)
+                && Enumerable.SequenceEqual(edges, other.edges)
+                && Enumerable.SequenceEqual(faces, other.faces);
```

Comment:
General question, how expensive is `SequenceEqual`? Is it notice-able in the profiler when you're moving around the scene?

Reply:
Just checked with the profiler and the operation does no GC Alloc and takes about 0.01ms. So it doesn't seem heavy at all.